admin/create-osx-user.sh

```diff
@@ -28,7 +28,7 @@ GROUP_ID=20
 ############### end of parameters
```

Comment:
Any changes to this file would need to be tested/approved as we are linking to this from the documentation for the customer to use.